site2/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "sync-request": "^6.1.0"
+  }
+}
```

Comment:
you should move this to website-next/package.json, because your scripts is in directory website-next @zeo1995